omc3/optics_measurements/crdt.py

```diff
@@ -47,23 +49,34 @@
 ]
 
 
-def calculate(measure_input, input_files, invariants, header):
-
+def calculate(measure_input: DotDict, input_files: InputFiles, invariants, header):
+    """ Calculate the CRDT values. 
+    
+    Todo: https://github.com/pylhc/omc3/issues/456 
```

Comment:
Since this PR fixes #456 , should we leave this as a todo? Maybe let's have the link into a comment somewhere in the body instead, having this show to the user is a bit ugly.

Reply:
This PR fixes the error, but will not close #456 , See also my comment in this PR note

Reply:
moved to the comments

Reply:
Good enough, thanks!